src/content/whats-new/2024/12/whats-new-12-03-fossa.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Gain security insights with FOSSA in New Relic vulnerability management dashboard'
```

Comment:
Check the capitalisation of the name of the dashboard in the UI

Reply:
Updated